--- gwenhywfar-3.8.3/configure.ac.wiget	2009-08-03 22:31:49.000000000 +0200
+++ gwenhywfar-3.8.3/configure.ac	2009-08-03 22:32:25.000000000 +0200
@@ -839,7 +839,7 @@ if test "$enable_visibility" = "yes"; th
   AC_MSG_CHECKING(if symbol visibility is supported)
   visibility_cflags=""
   case `$CC --version|head -n1|cut -f1 -d " "` in 
-    gcc*)
+    gcc*|*-gcc*)
        dnl Determine version number (watch out: the outside square
        dnl parenteses are vitally important here!)
        gccversion=["` ${CC} -dumpversion `"]
