--- gwenhywfar-4.3.0/src/sio/syncio_tls.c~	2011-10-02 15:15:58.000000000 +0200
+++ gwenhywfar-4.3.0/src/sio/syncio_tls.c	2011-10-02 15:49:18.044724558 +0200
@@ -470,7 +470,9 @@
   gnutls_transport_set_ptr(xio->session, (gnutls_transport_ptr_t)sio);
   gnutls_transport_set_push_function(xio->session, GWEN_SyncIo_Tls_Push);
   gnutls_transport_set_pull_function(xio->session, GWEN_SyncIo_Tls_Pull);
+#if GNUTLS_VERSION_NUMBER < 0x020c00
   gnutls_transport_set_lowat(xio->session, 0);
+#endif
 
   xio->prepared=1;
 
