--- gwenhywfar-2.4.1/configure.ac.orig	2006-09-26 11:05:18.000000000 +0200
+++ gwenhywfar-2.4.1/configure.ac	2006-12-04 01:16:35.675774827 +0100
@@ -582,10 +582,10 @@
 AC_MSG_CHECKING(if symbol visibility is supported)
 visibility_cflags=""
 case `basename $CC` in 
-  gcc*)
+  *-gcc*|gcc*)
      dnl Determine version number (watch out: the outside square
      dnl parenteses are vitally important here!)
-     gccversion=["` ${CC} --version | head -1 | ${SED} 's/.*\([0-9]\.[0-9]\.[0-9]*\).*/\1/' `"]
+     gccversion=["` ${CC} --version | head -n 1 | ${SED} 's/.*\([0-9]\.[0-9]\.[0-9]*\).*/\1/' `"]
      case $gccversion in
        4.*)
 	 # Activate -fvisibility=hidden only for explicit 4.x gcc
