--- gwenhywfar-3.0.0/configure.ac.orig	2007-11-21 19:59:52.000000000 +0100
+++ gwenhywfar-3.0.0/configure.ac	2007-12-16 22:28:05.674132044 +0100
@@ -788,10 +788,10 @@
   AC_MSG_CHECKING(if symbol visibility is supported)
   visibility_cflags=""
   case `basename $CC` in 
-    gcc*)
+    *-gcc*|gcc*)
        dnl Determine version number (watch out: the outside square
        dnl parenteses are vitally important here!)
-       gccversion=["` ${CC} --version | head -1 | ${SED} 's/.*\([0-9]\.[0-9]\.[0-9]*\).*/\1/' `"]
+       gccversion=["` ${CC} -dumpversion `"]
        case $gccversion in
          4.*)
            # Activate -fvisibility=hidden only for explicit 4.x gcc
