--- gwenhywfar-2.9.7beta/configure.ac.orig	2007-10-31 00:13:58.000000000 +0000
+++ gwenhywfar-2.9.7beta/configure.ac	2007-11-03 13:46:23.000000000 +0000
@@ -723,10 +723,10 @@
   AC_MSG_CHECKING(if symbol visibility is supported)
   visibility_cflags=""
   case `basename $CC` in 
-    gcc*)
+    *-gcc*|gcc*)
        dnl Determine version number (watch out: the outside square
        dnl parenteses are vitally important here!)
-       gccversion=["` ${CC} --version | head -1 | ${SED} 's/.*\([0-9]\.[0-9]\.[0-9]*\).*/\1/' `"]
+       gccversion=["` ${CC} --version | head -n 1 | ${SED} 's/.*\([0-9]\.[0-9]\.[0-9]*\).*/\1/' `"]
        case $gccversion in
          4.*)
            # Activate -fvisibility=hidden only for explicit 4.x gcc
